Sources/ApolloCodegenLib/FileGenerators/CustomScalarFileGenerator.swift

```diff
@@ -7,6 +7,6 @@ struct CustomScalarFileGenerator: FileGenerator {
 
   var template: TemplateRenderer { CustomScalarTemplate(graphqlScalar: graphqlScalar) }
   var target: FileTarget { .customScalar }
-  var fileName: String { "\(graphqlScalar.name.firstUppercased).swift" }
+  var fileName: String { "\(graphqlScalar.name).swift" }
```

Comment:
Why are we removing `.firstUppercased` here?

Reply:
Just because they were in each file generator implementation and now the protocol extension handles it. It does no harm here.

Reply:
I thought it would be clearer and more consistent if we had it the same in each of the file generators rather than asking ourselves questions later, when we forget this change, why some do it and others don't.

Reply:
Got it. Thanks!